auth/kerberos/src/krb.cpp

```diff
@@ -661,9 +662,13 @@ std::pair<int, std::string> get_gmsa_krb_ticket( std::string domain_name,
      *   Accounts,DC=contoso,DC=com' -s sub  "(objectClass=msDs-GroupManagedServiceAccount)"
      *   msDS-ManagedPassword
      */
+    std::string gmsa_ou = std::string( ",CN=Managed Service Accounts," );
+    if ( getenv(ENV_CF_GMSA_OU) != NULL)
+    {
+        gmsa_ou = std::string( "," ) + getenv(ENV_CF_GMSA_OU)+ std::string( "," );
```

Comment:
Would be good to have `std::string(getenv(ENV_CF_GMSA_OU))`, since getenv is a char *

Reply:
new cod push addressed the comment